website/docs/docs/cloud/about-cloud-develop-defer.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Using defer in dbt Cloud
+id: about-cloud-develop-defer
+description: "Learn how to leverage defer to prod when developing with dbt Cloud."
+sidebar_label: "Using defer in dbt Cloud"
+pagination_next: "docs/cloud/cloud-cli-installation"
+---
+
+
+[Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build these models' upstream parents. This is powered by having a production manifest to compare to, and dbt will resolve the `{{ ref() }}` function with upstream production artifacts.
+
+Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows, dramatically reducing development time and warehouse spend by preventing unecessary model builds. 
+
+## Required setup
+
+Set your production environment using the [production environment checkbox](/docs/deploy/deploy-environments#set-as-production-environment-beta) in the environment settings page. This can be set for one deployment environment per dbt Cloud project. When using defer, the artifacts from the latest successful, non-CI job execution in the production environment will be compared to.
+
+### Defer in the IDE
+
+Defer in the IDE is as simple as toggling the `Defer to production` switch on the command bar! When enabled, dbt Cloud will:
+
+1. Pull down the most recent manifest from the Production environment for comparison
+2. Pass the `--defer` flag to the command (for any command that accepts the flag)
```

Comment:
also working on clarifying what commands accepts what flags https://github.com/dbt-labs/docs.getdbt.com/issues/4206

Reply:
that would be a good example but the `--defer` and `--state` flags would be hidden to the user, so not sure we should include

Reply:
thanks so much for explaining! i'm just a little confused though - how can it be hidden for the user but we're saying they need to pass the `--defer` flag?

Reply:
it's supposed to indicate that dbt cloud will do the defer on their behalf! they won't need to pass the flag direectly, it will be auto-added in the background